src/99-appendices/08-coordinate-systems.md

```diff
@@ -50,16 +50,38 @@ information for a coordinate system (origin and orientation) are specified in
 `<datatype>CoordinateSystem`.
 The third piece of information for a coordinate system (units) are specified in
 `<datatype>CoordinateUnits`.
-Here, `<datatype>` can be one of `MEG`, `EEG`, or `iEEG`, depending on the
-modality that is being used.
+Here, `<datatype>` can be one of the following, depending on the data that
+is supposed to be documented:
+
+-   `MEG`
+-   `EEG`
+-   `iEEG`
+-   `Fiducials`
+-   `AnatomicalLandmark`
+-   `HeadCoil`
+-   `DigitizedHeadPoints`
 
 Allowed values for the `<datatype>CoordinateSystem` field come from a list of
 restricted keywords, as listed in the sections below.
+
+Note that `Fiducials`, `AnatomicalLandmark`, `HeadCoil`, and `DigitizedHeadPoints`
+datatypes share the restricted keywords with the data modality they are shared with.
+For example, if a `AnatomicalLandmark` field is shared as part of an EEG dataset,
+the EEG specific coordinate systems apply.
+However, if it is shared as part of an MEG dataset,
+the EEG specific coordinate systems apply.
+
 If no value from the list of restricted keywords fits, there is always the
 option to specify the value as follows:
 
--   `Other`: Use this for other coordinate systems and specify further details
-    in the `<datatype>CoordinateSystemDescription` field
+-   `Other`: Use this value for other coordinate systems and specify all
+    required details in the `<datatype>CoordinateSystemDescription` field
+
+-   `Image`: Use this value in conjunction with the `IntendedFor` field.
+    For example, if all coordinates you supply are to be interpreted with
+    respect to a specific anatomical T1w image, specify `<datatype>CoordinateSystem`
+    as `Image`, and `IndendedFor` with the path to that image in the
```

Comment:
```suggestion
    as `Image`, and `IntendedFor` with the path to that image in the
```

New fields shouldn't be described only in appendices. This also seems like a reversal of the usual way that `IntendedFor` is used. The fieldmap is an adjunct to the image it is intended for. Here the anatomical appears to be the adjunct of the MEG/EEG data that are "intended for" it. What about co-opting the `SpatialReference` field from derivatives?